codegen-maven/src/main/java/com/paypal/selion/plugins/CodeGeneratorMojoLogger.java

```diff
@@ -0,0 +1,127 @@
+/*-------------------------------------------------------------------------------------------------------------------*\
+|  Copyright (C) 2017 PayPal                                                                                          |
+|                                                                                                                     |
+|  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance     |
+|  with the License.                                                                                                  |
+|                                                                                                                     |
+|  You may obtain a copy of the License at                                                                            |
+|                                                                                                                     |
+|       http://www.apache.org/licenses/LICENSE-2.0                                                                    |
+|                                                                                                                     |
+|  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed   |
+|  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for  |
+|  the specific language governing permissions and limitations under the License.                                     |
+\*-------------------------------------------------------------------------------------------------------------------*/
+
+package com.paypal.selion.plugins;
+
+import com.google.common.annotations.VisibleForTesting;
+import org.apache.maven.plugin.logging.Log;
+import org.apache.maven.plugin.logging.SystemStreamLog;
+
+public class CodeGeneratorMojoLogger implements CodeGeneratorLogger, Log {
+    private final Log instance;
+
+    public CodeGeneratorMojoLogger() {
+        this.instance = new SystemStreamLog();
+    }
+
+    public CodeGeneratorMojoLogger(Log instance) {
+        this.instance = instance;
+    }
```

Comment:
The reason the class `CodeGeneratorMojoLogger` exists is because `Log` is an interface in a maven plugin artifact dependency. I didn't want the SeLion Code Generator CLI to have any maven artifact dependencies. So, the CLI version defines the interface `CodeGeneratorLogger` ...

This class bridges `CodeGeneratorLogger` and `Log` so  the SeLion Code Generator maven plugin can have a`Log` and the code it depends on (SeLion Code Generator CLI) can have a `CodeGeneratorLogger` -- both as the same logger instance.